opennms-base-assembly/src/main/filtered/etc/snmp-graph.properties.d/didactum-graph.properties

```diff
@@ -0,0 +1,58 @@
+##############################################################################
+##
+##  Please add report definition in a new line to make it easier
+##  for script based sanity checks
+##
+##################################################
+
+reports=DidactumMonitoringSystem 
```

Comment:
to be more compliant with the current configs, I would suggest to rename to `didactum.monitoringSystem`